packages/core/src/entities/Repo.ts

```diff
@@ -1,30 +1,69 @@
-import Entity from "./Entity"
-import { ConnectorInterface } from "../connections/ConnectorInterface"
+import Entity from './Entity'
+import {
+  AragonArtifact,
+  AragonManifest,
+  AragonEnvironments,
+  AragonArtifactRole,
+} from '../types'
+import { ConnectorInterface } from '../connections/ConnectorInterface'
 
 // TODO: Implement all properties and methods from the API spec (https://github.com/aragon/plumbery/blob/master/docs/repo.md).
-// [ ] author 	String 	Author of the app. E.g. "Aragon Association".
 // [ ] changelogUrl 	String 	URL of the app versions changelog.
```

Comment:
Was not sure about this value, where is this changelog?

Reply:
In the manifest: https://github.com/aragon/aragon-apps/blob/master/apps/token-manager/manifest.json#L5